doc-files/release-notes-21.md

```diff
@@ -0,0 +1,153 @@
+# Release Notes for JavaFX 21
+
+## Introduction
+
+The following notes describe important changes and information about this release. In some cases, the descriptions provide links to additional detailed information about an issue or a change.
+
+These release notes cover the standalone JavaFX 21 release. JavaFX 21 requires JDK 17 or later.
+
+## Important Changes
+
+### JavaFX Requires macOS 11 or Later
+
+On Mac platforms, JavaFX 21 requires macOS 11 or later. An exception will be thrown when initializing the JavaFX runtime on older versions of macOS.
+
+See [JDK-8308114](https://bugs.openjdk.org/browse/JDK-8308114) for more information.
+
+### JavaFX Requires GTK 3 on Linux
+
+On Linux platforms, JavaFX 21 requires GTK3 version 3.8 or later. The JavaFX GTK2 library has been removed. An exception will be thrown when initializing the JavaFX runtime if the GTK 3 library cannot be loaded.
+
+See [JDK-8299595](https://bugs.openjdk.org/browse/JDK-8299595) for more information.
+
+### Return Type of javafx.css.Match::getPseudoClasses Changed
+
+The return type of `javafx.css.Match.getPseudoClasses` has been changed to `Set<PseudoClass>` instead of a non-public type. Most applications will not be impacted by this change, since this is not a method that applications ever need to call. In the unlikely event that an existing application binary does call this method, a `NoSuchMethodError` will be thrown. The solution is to recompile the application.
+
+See [JDK-8304959](https://bugs.openjdk.org/browse/JDK-8304959) for more information.
+
+### Event Handler Methods Added to EventTarget Interface
+
+Four new default methods were added to the `javafx.event.EventTarget` interface:
+
+```
+<E extends Event> void addEventHandler(EventType<E>, EventHandler<? super E>)
+<E extends Event> void removeEventHandler(EventType<E>, EventHandler<? super E>)
+<E extends Event> void addEventFilter(EventType<E>, EventHandler<? super E>)
+<E extends Event> void removeEventFilter(EventType<E>, EventHandler<? super E>)
+```
+
+This change allows applications to manage event handlers for all `EventTarget` implementations.
+
+Most applications will not run into any backward compatibility problems with this change. The change is fully binary compatible and, for most use cases, it is also source compatible.
+
+A compilation error will occur in one of the following two unlikely cases:
+
+* An application class extends `Menu`, `MenuItem`, `TreeColumnBase`, or `TreeItem`, and overrides `addEventHandler` or `removeEventHandler`
```

Comment:
hmm .. did I miss a new class (TreeColumnBase), or shouldn't that be TableColumnBase?

Reply:
Well how embarrassing! Thanks for pointing out the typo that none of the rest of us saw. In addition to fixing the release notes, I'll update the Risk and Solution sections of the CSR, which is where the typo originated (the Specification section of the CSR has it right).